bitnami/pytorch/templates/service.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "pytorch.fullname" . }}
+  labels:
+    app.kubernetes.io/name: {{ include "pytorch.name" . }}
+    helm.sh/chart: {{ include "pytorch.chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/component: "master"
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+    - port: {{ .Values.service.port }}
+      targetPort: pytorch
+      name: pytorch
```

Comment:
`nodePort` should be configurable when `service.type = "NodePort"`

Reply:
I don't think it makes senses anything other than `ClusterIP`.